php-fpm/docker-entrypoint.sh

```diff
@@ -23,7 +23,7 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 	fi
 
 	if [ "$APP_ENV" != 'prod' ]; then
-		composer install --prefer-dist --no-progress --no-suggest --no-interaction
+		composer install --prefer-dist --no-progress --no-interaction
```

Comment:
I see `no-suggest` option is missing in v2, Isn't it? Good!!

Reply:
yes, that's why I removed it